.github/workflows/gradle.yml

```diff
@@ -23,11 +23,11 @@ jobs:
     - name: Set up JDK
       uses: actions/setup-java@v3
       with:
-        java-version: '18'
+        java-version: '24'
```

Comment:
Hmm, JDK 24 seems to have a very short support window, with support already ending this year. Wouldn't it make more sense to go for one of the LTS versions, e.g., 21 or 25 when it comes out later this year?